CMakeLists.txt

```diff
@@ -28,33 +29,41 @@ endif()
 if(WIN32)
     set(LAPACK_DIR ${THIRD_PARTY_DIR}/suitesparse_source/lapack_windows/x${OS_BITS})
 endif()
+
+file(DOWNLOAD ${suitesparse_url} ${CMAKE_CURRENT_BINARY_DIR}/${suitesparse_archive})
+
 ExternalProject_Add(suitesparse
     SOURCE_DIR ${THIRD_PARTY_DIR}/suitesparse_source
     GIT_REPOSITORY "https://github.com/jlblancoc/suitesparse-metis-for-windows.git"
-    GIT_TAG "master"
-    PATCH_COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/suitesparse_patch.py ${suitesparse_version}
+    GIT_TAG "v1.5.0"
+    PATCH_COMMAND cmake -E tar xzf ${CMAKE_CURRENT_BINARY_DIR}/${suitesparse_archive}
```

Comment:
Nice we can get rid of this python script!

Reply:
Indeed! I discover this feature this morning. Cmake support tar command natively so I chose to use it and to remove our dependency to python.